examples/bri-2/ops/check_dot_env.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+FILE="./commit-mgr/.env"
```

Comment:
Use `pwd` convention?

Reply:
Please excuse my ignorance @skarred14 but what is `pwd` convention?

Reply:
"print working directory" -  but this path is relative so perhaps it already conforms to that convention :)

Reply:
Yes I am familiar with the `pwd` acronym and command. I am just not sure what "convention" refers to in this context.